spec/system/grant_submissions/grant_submission_submissions_spec.rb

```diff
@@ -194,41 +190,62 @@
 
       context '#update' do
         before(:each) do
-          @submission = create(:draft_submission_with_responses,
-                              grant:      @grant,
-                              form:       @grant.form,
-                              created_id: @applicant.id,
-                              title:      Faker::Lorem.sentence,
-                              state:      'draft')
-
-          login_as(@applicant)
+          submission.update_attribute(:state, 'draft')
+          login_as(applicant)
         end
 
-        scenario 'can visit edit path for draft submission' do
-          visit edit_grant_submission_path(@grant, @submission)
-          expect(page).to have_content "Editing Application"
-          expect(page).to have_current_path edit_grant_submission_path(@grant, @submission)
+        context 'draft submission' do
+          scenario 'can visit edit path for draft submission' do
+            visit edit_grant_submission_path(grant, submission)
+            expect(page).to have_content 'Editing Application'
+            expect(page).to have_current_path edit_grant_submission_path(grant, submission)
+          end
+
+          scenario 'can save submission as draft' do
+            visit edit_grant_submission_path(grant, submission)
+            find_field('Short Text Question').set(Faker::Lorem.sentence)
```

Comment:
in this scenario does the grant have any required question? it should have at least one to verify that a draft can be saved without filling it